src/qt/overviewpage.cpp

```diff
@@ -180,7 +180,10 @@ void OverviewPage::setBalance(const interfaces::WalletBalances& balances)
     m_balances = balances;
     if (walletModel->wallet().isLegacy()) {
         if (walletModel->wallet().privateKeysDisabled()) {
+            ui->labelSpendable->setText(QApplication::translate("OverviewPage", "Watch-only:", Q_NULLPTR));
+            ui->labelSpendable->setVisible(true);
             ui->labelBalance->setText(BitcoinUnits::formatWithPrivacy(unit, balances.watch_only_balance, BitcoinUnits::SeparatorStyle::ALWAYS, m_privacy));
+            ui->labelBalance->setToolTip(QApplication::translate("OverviewPage", "Your current balance in watch-only addresses", Q_NULLPTR));
```

Comment:
```suggestion
            ui->labelBalance->setToolTip(tr("Your current balance in watch-only addresses"));
```

Reply:
effect is the the same, but to translate her by tr makes sense since we have that in the object, and fever code is better code ;-)